tests/unit_tests_src/for_2D_build/geometies/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+SUBDIRLIST(SUBDIRS ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Name of the folder is misspelt : `geometies` &rarr; `geometries`

Reply:
Ok. It will updated in the merge.